config/spryk/spryk.yml

```diff
@@ -1,6 +1,12 @@
 arguments:
+  sprykerOrganization:
+      value: "spryker/"
+  sprykerBundles:
+      value: "Bundles/"
   repository:
-    value: "spryker/{{ organization | dasherize }}"
+    value: "{{ sprykerOrganization }}{{ organization | dasherize }}"
```

Comment:
did not get what is organization here?

something is vendor name = organization (spryker, spryker-shop, spryker-eco, custom)
and another one is package (product, logger, ...)

Reply:
Just my thoughts (let's discuss if it's possible/not possible to change):

in case of Spryker.A or SprykerShop.A

```
organization = spryker
package = spryker or spryker-shop
directory = Bundles/
module = A
```

Reply:
> did not get what is organization here?

organization - it is Spryker / SprykerShop / SprykerEco etc

sprykerOrganization - its only spryker/ on nothing for eco sdk, etc

Reply:
organization its actually one of  CORE_NAMESPACE values,